ods_ci/tests/Resources/Common.robot

```diff
@@ -496,3 +496,12 @@ Clear Element And Input Text
     Clear Element Text    ${element_xpath}
     Sleep    0.5s
     Input Text    ${element_xpath}    ${new_text}
+
+Wait For Dashboard Page Title
```

Comment:
maybe this could have gone in `ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDashboard.robot`

Reply:
Thanks @bdattoma, moved to ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDashboard.robot